ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent-R2-CUI-nonprod.json

```diff
@@ -479,6 +479,44 @@
       }
     ]
   },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "NOTIFY_LIP_RESPONDENT_CLAIMANT_CONFIRM_TO_PROCEED",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "caseworker-civil-systemupdate"
+        ],
+        "CRUD": "CRU"
+      },
+      {
+        "UserRoles": [
+          "CITIZEN-CLAIMANT-PROFILE",
+          "CITIZEN-DEFENDANT-PROFILE"
+        ],
+        "CRUD": "R"
+      }
+    ]
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "NOTIFY_LIP_APPLICANT_CLAIMANT_CONFIRM_TO_PROCEED",
```

Comment:
Same here, if it notifies only applicant there is no need to say claimant in the event?

Reply:
Same here
I used the same format to name this one (notification for the applicant) because I understood that the intention is to specify that these two events happen when the CLAIMANT_CONFIRM_TO_PROCEED, one event to NOTIFY_LIP_APPLICANT and the other to NOTIFY_LIP_RESPONDENT